service/frontend/workflowHandler.go

```diff
@@ -2489,6 +2489,7 @@ func (wh *WorkflowHandler) ListOpenWorkflowExecutions(ctx context.Context,
 	resp = &gen.ListOpenWorkflowExecutionsResponse{}
 	resp.Executions = persistenceResp.Executions
 	resp.NextPageToken = persistenceResp.NextPageToken
+	checkAndSetExecutionTime(resp.Executions)
```

Comment:
This is something I am not sure. I originally did this check in persistence layer, but I thought it will be better to perform the check in one place, instead of multiple place, one for each persistence component, which is hard to manage and update. Your comment also makes a lot of sense. I will change it back.